codalab/lib/bundle_cli.py

```diff
@@ -1455,43 +1455,50 @@ def copy_bundle(
             },
         )
 
-        # If bundle contents don't exist, finish after just copying metadata
+        # Fetch bundle contents from source client
```

Comment:
This is not fetching bundle contents, but just the metadata

Reply:
Updated.